tests/unit/app/components/nf-y-axis-test.js

```diff
@@ -1,67 +1,29 @@
-import Ember from 'ember';
-import {
-  moduleForComponent,
-  test
-} from 'ember-qunit';
-
+import { moduleForComponent, test } from 'ember-qunit';
+import { run } from '@ember/runloop';
 
 moduleForComponent('nf-y-axis', {
-  // specify the other units that are required for this test
+  unit: true,
   needs: ['component:nf-graph']
 });
 
 test('nf-y-axis tickData should call tickFactory if available', function(assert) {
   var args;
-  Ember.run(() => {
-    //HACK: couldn't find a great, documented way of mocking readonly and aliased propertied
-    // so I override them here.
-    var axis = this.factory().extend({
-      uniqueYData: [1,2,3,4,5],
+  run(() => {
+    var axis = this.subject({
       yScale: 'yScale',
-      graph: Ember.computed((key, value) => ({
-        yScaleType: 'yScaleType'
-      })),
+      graph: {
+        yScaleType: 'yScaleType',
+        yData: [1,2,3,4,5]
+      },
       tickCount: 42,
-      tickFactory: function() {
+      tickFactory() {
         args = [].slice.call(arguments);
         return 'expected result';
       }
-    }).create();
+    });
 
     var tickData = axis.get('tickData');
     assert.equal(tickData, 'expected result');
     assert.deepEqual(args, ['yScale', 42, [1,2,3,4,5], 'yScaleType']);
   });
 });
-
```

Comment:
These (and the similar ones in `nf-x-axis`) are the tests that didn't seem necessary anymore. It looks like testing Ember internals to me.

If they are still necessary, it would make more sense to write Integration-style tests that ensure that the expected behavior happens when used with and without a block.